src/schematools/cli.py

```diff
@@ -325,8 +325,11 @@ def _fetch_json(location: str) -> Dict[str, Any]:
 @schema.command()
 @option_schema_url
 @argument_schema_location
+@click.option("--additional-schemas", "-a", multiple=True)
```

Comment:
This parameter ` should either be documented with an extra `help` argument, or `validate`'s docstring should be amended with info about this new functionality/parameter

Reply:
Good point, I'll add a help argument.

Reply:
Do I understand correctly that the syntax would be

`--additional-schemas=foo --additional-schemas=bar`

? If so, I think the option name should be singular. Using a plural suggests that the argument itself is a list.